### ISSUE TITLE
Add osx-arm64 build

### ISSUE THREAD

**Issue by tomwhite** [conda-forge/tskit-feedstock#25]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

This is needed to install on a M1 Mac Mini. There are some instructions here: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : sgkit-dev-3.8
    active env location : /opt/homebrew/Caskroom/miniforge/base/envs/sgkit-dev-3.8
            shell level : 2
       user config file : /Users/tom/.condarc
 populated config files : /opt/homebrew/Caskroom/miniforge/base/.condarc
          conda version : 4.10.3
    conda-build version : not installed
         python version : 3.9.6.final.0
       virtual packages : __osx=11.5.2=0
                          __unix=0=0
                          __archspec=1=arm64
       base environment : /opt/homebrew/Caskroom/miniforge/base  (writable)
      conda av data dir : /opt/homebrew/Caskroom/miniforge/base/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /opt/homebrew/Caskroom/miniforge/base/pkgs
                          /Users/tom/.conda/pkgs
       envs directories : /opt/homebrew/Caskroom/miniforge/base/envs
                          /Users/tom/.conda/envs
               platform : osx-arm64
             user-agent : conda/4.10.3 requests/2.26.0 CPython/3.9.6 Darwin/20.6.0 OSX/11.5.2
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```
</details>

**Comment by jeromekelleher** [comment]
Just merged #26 which *should* fix this @tomwhite - would you mind trying a tskit install please and reporting back?

**Comment by tomwhite** [comment]
It works (I can run `tskit --version` at least). Thanks for the quick turnaround!
